capstone/src/main/webapp/explore-loader.js

```diff
@@ -60,9 +61,13 @@ function getLabelQueryString () {
   return queryString.slice(0, -1); // Takes out the last comma. 
 }
 
-function getJoinButton(clubName) {
-  var joinButton = '<button name="join" value="'
-    + clubName
-    + '" formmethod="POST">Join Club</button>';
-  return joinButton;
-}
\ No newline at end of file
+function sendJoinedAlert(interestedOrJoin) {
+  var alertMessage = 'You have successfully ';
+  if (interestedOrJoin == 'interested') {
+    alertMessage += 'expressed interest in this club! You can view your interested club list on your profile page.'
+  }
+  else {
```

Comment:
super nit: single line `} else {`

https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style
"For example, there is no line break after the brace if it is followed by else or a comma."